src/core/lib/iomgr/exec_ctx.cc

```diff
@@ -140,7 +140,7 @@ static gpr_atm timespec_to_atm_round_up(gpr_timespec ts) {
 
 grpc_millis grpc_exec_ctx_now(grpc_exec_ctx *exec_ctx) {
   if (!exec_ctx->now_is_valid) {
-    exec_ctx->now = timespec_to_atm_round_down(gpr_now(GPR_CLOCK_MONOTONIC));
+    exec_ctx->now = timespec_to_atm_round_up(gpr_now(GPR_CLOCK_MONOTONIC));
```

Comment:
This will make deadlines expire before they are supposed to, correct?

Reply:
Right... pretty sure we don't want that